packages/ui-default/templates/contest_edit.html

```
@@ -79,6 +79,22 @@ <h2 class="section__title">{{ _('Basic Info') }}</h2>
+          <div class="row">
+            {{ form.form_checkbox({
+              columns:9,
+              label:'Limit language',
+              name:'limitLang',
+              placeholder:_('Limit contest submission language'),
+              value:isLimitLang,
+              row:false,
+              disabled:true
+            }) }}
+          </div>
+          <div id="language-select-part" {% if not isLimitLang %}style="display:none;"{% endif %}>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)